models/resnet/tensorflow2/train_tf2_resnet.py

```diff
@@ -136,7 +136,7 @@ def main():
     model = tf.keras.applications.ResNet50(weights=None, classes=1000)
     opt = tf.keras.optimizers.SGD(learning_rate=FLAGS.learning_rate * hvd.size(), momentum=0.1)
     if not FLAGS.fp32:
-        opt = tf.keras.mixed_precision.experimental.LossScaleOptimizer(opt, loss_scale="dynamic")
```

Comment:
Get the TF version and make the change condition of TF > 2.9

Reply:
Fixed